<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0" stdsetdef="1">
  <author></author>
  <comment></comment>
  <exportmacro></exportmacro>
  <class>XxSearchDialogBase</class>
  <widget class="QDialog" name="XxSearchDialogBase">
    <property name="geometry">
      <rect>
        <x>0</x>
        <y>0</y>
        <width>619</width>
        <height>232</height>
      </rect>
    </property>
    <property name="windowTitle">
      <string>Search</string>
    </property>
    <property name="sizeGripEnabled">
      <bool>true</bool>
    </property>
    <layout class="QVBoxLayout">
      <property name="margin">
        <number>11</number>
      </property>
      <property name="spacing">
        <number>6</number>
      </property>
      <item>
        <layout class="QHBoxLayout">
          <property name="margin">
            <number>0</number>
          </property>
          <property name="spacing">
            <number>6</number>
          </property>
          <item>
            <widget class="QLabel" name="TextLabel1">
              <property name="text">
                <string>Search string:</string>
              </property>
              <property name="wordWrap">
                <bool>false</bool>
              </property>
            </widget>
          </item>
          <item>
            <widget class="QLineEdit" name="_lineeditSearchString"/>
          </item>
        </layout>
      </item>
      <item>
        <widget class="QGroupBox" name="GroupBox1">
          <property name="title">
            <string>Options</string>
          </property>
          <layout class="QHBoxLayout">
            <property name="margin">
              <number>11</number>
            </property>
            <property name="spacing">
              <number>6</number>
            </property>
            <item>
              <widget class="QCheckBox" name="_checkboxCaseSensitive">
                <property name="enabled">
                  <bool>false</bool>
                </property>
                <property name="text">
                  <string>Case sensitive</string>
                </property>
              </widget>
            </item>
            <item>
              <widget class="QCheckBox" name="_checkboxUseRegexp">
                <property name="enabled">
                  <bool>false</bool>
                </property>
                <property name="text">
                  <string>Regexp search</string>
                </property>
              </widget>
            </item>
          </layout>
        </widget>
      </item>
      <item>
        <layout class="QHBoxLayout">
          <property name="margin">
            <number>0</number>
          </property>
          <property name="spacing">
            <number>6</number>
          </property>
          <item>
            <widget class="QPushButton" name="_buttonApply">
              <property name="text">
                <string>&amp;Apply</string>
              </property>
              <property name="autoDefault">
                <bool>true</bool>
              </property>
              <property name="default">
                <bool>true</bool>
              </property>
            </widget>
          </item>
          <item>
            <widget class="QPushButton" name="_buttonPrevious">
              <property name="text">
                <string>&lt;&lt; Previous</string>
              </property>
            </widget>
          </item>
          <item>
            <widget class="QPushButton" name="_buttonNext">
              <property name="text">
                <string>Next &gt;&gt;</string>
              </property>
            </widget>
          </item>
          <item>
            <spacer name="Horizontal Spacing2">
              <property name="sizeHint">
                <size>
                  <width>20</width>
                  <height>20</height>
                </size>
              </property>
              <property name="sizeType">
                <enum>QSizePolicy::Expanding</enum>
              </property>
              <property name="orientation">
                <enum>Qt::Horizontal</enum>
              </property>
            </spacer>
          </item>
          <item>
            <widget class="QPushButton" name="_buttonClose">
              <property name="text">
                <string>&amp;Close</string>
              </property>
              <property name="autoDefault">
                <bool>true</bool>
              </property>
            </widget>
          </item>
        </layout>
      </item>
      <item>
        <layout class="QHBoxLayout">
          <property name="margin">
            <number>0</number>
          </property>
          <property name="spacing">
            <number>6</number>
          </property>
          <item>
            <widget class="QLabel" name="TextLabel1_2">
              <property name="text">
                <string>Goto line:</string>
              </property>
              <property name="wordWrap">
                <bool>false</bool>
              </property>
            </widget>
          </item>
          <item>
            <widget class="QLineEdit" name="_lineeditGotoLine">
              <property name="text">
                <string>1</string>
              </property>
            </widget>
          </item>
          <item>
            <widget class="QComboBox" name="_comboGotoWhichFile">
              <item>
                <property name="text">
                  <string>left</string>
                </property>
              </item>
              <item>
                <property name="text">
                  <string>right</string>
                </property>
              </item>
              <item>
                <property name="text">
                  <string>middle</string>
                </property>
              </item>
            </widget>
          </item>
          <item>
            <widget class="QPushButton" name="_buttonGotoLine">
              <property name="text">
                <string>&gt;&gt; Goto line</string>
              </property>
            </widget>
          </item>
          <item>
            <spacer name="Spacer2">
              <property name="sizeHint">
                <size>
                  <width>20</width>
                  <height>20</height>
                </size>
              </property>
              <property name="sizeType">
                <enum>QSizePolicy::Expanding</enum>
              </property>
              <property name="orientation">
                <enum>Qt::Horizontal</enum>
              </property>
            </spacer>
          </item>
        </layout>
      </item>
    </layout>
  </widget>
  <connections>
    <connection>
      <sender>_buttonClose</sender>
      <signal>clicked()</signal>
      <receiver>XxSearchDialogBase</receiver>
      <slot>reject()</slot>
    </connection>
  </connections>
</ui>
